test/e2e/page.spec.js

```diff
@@ -63,7 +63,7 @@ test.describe('page', () => {
 		});
 
 		test('should contain 14 top level ba-components', async ({ page }) => {
-			expect(await page.locator('body > *').count()).toBe(14);
+			expect(await page.locator('body > *').count()).toBe(15);
```

Comment:
a specific test for the component is missing.

```suggestion
			expect(await page.locator('ba-navigation-rail').count()).toBe(1);
```